Python/pywarpx/picmi.py

```diff
@@ -402,34 +406,6 @@ def initialize_inputs(self, species_number, layout, species, density_scale):
         else:
             species.__setattr__('density_function(x,y,z)', "{}*({})".format(density_scale, expression))
```

Comment:
This could lead to issues in cases where people are using the parsers ability to calculate sequential variable values, for example:
`density_function(x,y,z) = "r=sqrt(x**2+y**2);1e15*(r < 50e-6)"`
Maybe we should remove the `density_scale` parameter when a density function is used?

Reply:
Good point. This was written before that type of expression was allowed. Presumably the resulting expression like this is not valid
`3*(r=sqrt(x**2+y**2);1e15*(r < 50e-6))`

Reply:
Oh cool! That is good to know. Thanks for testing.